README.rst

```diff
@@ -96,8 +96,28 @@ The icon picker supports `numerous font libraries <http://victor-valencia.github
 out of the box. You can also add multiple font sets like this::
 
     DJANGOCMS_ICON_SETS = [
-        ('fontawesome5', 'fa', 'Font Awesome'),
+        ('elusiveicon', 'el', 'Elusive Icons'),
+        ('flagicon', 'flag-icon', 'Flag Icons'),
+        ('fontawesome4', 'fa', 'Font Awesome 4'),
+        ('fontawesome5regular', 'far', 'Font Awesome 5 Regular'),
```

Comment:
Given that most users will probably use the free version, wouldn't it make more sense to add "fontawesome5" and "fontawesome5pro" as choices to simplify the setup?

Reply:
I had to split the icons into separate sets to enable the preselection of the correct icon set and icon in the icon selector widget. Since there is no other context available in the widget the `iconClass` in the set has to be available and it has to be unique in the settings.py as well. If you look at the fields.py you can see why I did this. Maybe you have a better idea circumvent this issue?